src/components/ProgressBar/ProgressBar.js

```diff
@@ -50,8 +50,12 @@ export const ProgressBar = () => {
     }
 
     if (status === JOB_STATUSES.FAILED) {
-      swwCallout();
-      clearIntervalAndRedirect('/bulk-edit', '');
+      if (bulkDetails?.approach === APPROACHES.QUERY) {
```

Comment:
I can't agree, what if query job just failed?

Reply:
added BadRequest substing as errorSign. Will show callout in such cases